style.css

```diff
@@ -26,6 +26,53 @@ body {
   height: 30px;
 }
 
+.menuList {
```

Comment:
- [ ] like the previous reviewer noted, when your menu comes up it should be used as a means of navigation to the other page sections. The user should not be able to scroll through your page unless he/she uses the menu links.
![demo](https://user-images.githubusercontent.com/65298048/179732067-331e8a94-623e-4214-8d62-ee7230eace4a.gif)

Reply:
Changes has been made to correct the errors,

thank you for the kind review.